rust/src/protocol/parquet2_read/mod.rs

```diff
@@ -64,6 +64,40 @@ struct DeserState {
     metadata_configuration: map::MapState,
 }
 
+impl From<String> for ReaderFeatures {
+    fn from(value: String) -> Self {
+        match value.as_str() {
+            "columnMapping" => ReaderFeatures::COLUMN_MAPPING,
+            "deletionVectors" => ReaderFeatures::DELETION_VECTORS,
+            "timestampNtz" => ReaderFeatures::TIMESTAMP_WITHOUT_TIMEZONE,
+            "v2Checkpoint" => ReaderFeatures::V2_CHECKPOINT,
+            f => panic!("Unknown reader feature encountered: {}", f),
```

Comment:
I don't think this should warrant a panic. Just unsupported table error. I'd convert this into some `Other` variant and maybe let the error be handled elsewhere.

Reply:
+1 on the not having these functions panic.